### PR TITLE
Use MOJ Design System (remove GOV.UK logo)

### PR DESCRIPTION
**Pull request by andyhd**
* Also remove HMCTS Design System
* Change logout view to accept `GET` request and change disguised form button to link in header
* Fix references to deprecated GOV.UK Frontend colours

### CODE REVIEW

File: controlpanel/frontend/static/components/navbar/macro.html [hunk 1]
```diff
@@ -1,19 +1,37 @@
...
-         {%- for key, value in item.attributes %} {{key}}="{{value}}"
-         {%- endfor %}>{{ item.html | safe if item.html else item.text }}</a>
-    </li>
-    {% endif %}
-  {% endfor %}
-  </ul>
-</nav>
+{% macro mojPrimaryNavigation(params) %}
+<div class="moj-primary-navigation {{- ' ' + params.containerClasses if params.containerClasses else ''}}">
+
```
Comment (s-block) [inline controlpanel/frontend/static/components/navbar/macro.html]: Does this need so many blank lines?

File: controlpanel/frontend/jinja2/base.html [hunk 2]
```diff
@@ -36,31 +35,48 @@
...
+        'label': 'Account navigation',
+        'items': [
+          {
+            'href': url('index'),
+            'text': request.user.name | default(request.user.email),
+            'active': True
+          },
+          {
+            'html': '<a href="' ~ url('oidc_logout') ~ '">Sign out</a>'
+,
```
Comment (s-block) [inline controlpanel/frontend/jinja2/base.html]: Does this need to be here?

File: controlpanel/frontend/static/components/navbar/macro.html (hunk 1 above)
Comment (andyhd) [reply controlpanel/frontend/static/components/navbar/macro.html]: This is copied from [MOJ Frontend](https://github.com/ministryofjustice/moj-frontend/blob/master/src/moj/components/primary-navigation/template.njk) with minor changes to make it work with Jinja. Probably the fewer differences, the better.

File: controlpanel/frontend/jinja2/base.html (hunk 2 above)
Comment (andyhd) [reply controlpanel/frontend/jinja2/base.html]: Oops, typo

File: controlpanel/frontend/static/components/navbar/macro.html (hunk 1 above)
Comment (s-block) [reply controlpanel/frontend/static/components/navbar/macro.html]: :+1:

**Pull request merged**